devops/scripts/benchmarks/benches/compute.py

```diff
@@ -560,9 +570,10 @@ def extra_env_vars(self) -> dict:
 
     def name(self):
         return (
-            f"multithread_benchmark_ur MemcpyExecute opsPerThread:{self.numOpsPerThread}, numThreads:{self.numThreads}, allocSize:{self.allocSize} srcUSM:{self.srcUSM} dstUSM:{self.dstUSM}"
+            f"multithread_benchmark_{self.runtime.value} MemcpyExecute opsPerThread:{self.numOpsPerThread}, numThreads:{self.numThreads}, allocSize:{self.allocSize} srcUSM:{self.srcUSM} dstUSM:{self.dstUSM}"
             + (" without events" if not self.useEvents else "")
             + (" without copy offload" if not self.useCopyOffload else "")
+            + (" with barrier" if self.useBarrier else "")
         )
 
     def explicit_group(self):
```

Comment:
these will end up being in the same group. Is that intentional? I think you might also want to include `allocSize` as part of the group, since SYCL_PREVIEW and UR benchmarks differ there.

Reply:
There's quite a few of this benchmarks so I think having them together will help with analyzing. Good point about the allocSize, done.